### ISSUE TITLE
README.md composition

### ISSUE THREAD

**Issue by judemin** [euphony-io/sound-helper#2]
앱 사용법과 각 페이지의 목적등을 간결하게 정리
영어, 한국어 모두 작성
샘플 애플리케이션 개발 후 작성

**Comment by kuro11pow2** [comment]
![chrome_ebpQdqropn](https://user-images.githubusercontent.com/47289893/129519279-7163372f-ebaa-49d4-9191-97156817dab9.png)
![chrome_MwPGcI4955](https://user-images.githubusercontent.com/47289893/129519282-000e179e-04d4-4ff2-b081-930dfa5bc15e.png)

How about writing two documents in English and Korean like the attached image?

**Comment by judemin** [comment]
I agree. Like early reviews of mentors in euphony-listener we could write two readme files.
(README.md, README.kr.md)

**Comment by kuro11pow2** [comment]
All right, let's talk about the composition of README.md. I think it would be good to put "the objectives of app" and "how to achieve objectives" in the "Introduction" section.

**Comment by kmsaidev** [comment]
Also, I think it would be good to write quick-start through pictures like euphony-listener. To make it easier for people who are new to use it!

**Comment by kuro11pow2** [comment]
> Also, I think it would be good to write quick-start through pictures like euphony-listener. To make it easier for people who are new to use it!

I agree with your opinion!

**Comment by judemin** [comment]
How about using this template for our README?
https://github.com/always0ne/repositoryTemplate

**Comment by eunzinn** [comment]
> How about using this template for our README?
> https://github.com/always0ne/repositoryTemplate

Good! Using an index is a good idea.

**Comment by kuro11pow2** [comment]
> How about using this template for our README?
> https://github.com/always0ne/repositoryTemplate

The template looks good! I agree to use this template.

**Comment by judemin** [comment]
Just created README.md only with template.
Maybe we should divide roles as we did in euphony-listener.

**Comment by eunzinn** [comment]
> Just created README.md only with template.
> Maybe we should divide roles as we did in euphony-listener.

Do we have to make the configuration(overview, getting stated, contributing ...) as it is in the template?

**Comment by judemin** [comment]
Yes, I think we should add some androidTestImplementation, modules for readers.
